draft-ietf-quic-multipath.md

```diff
@@ -357,7 +357,7 @@ When more than one path is available, hosts shall monitor the arrival
 of non-probing packets  and the acknowledgements 
 for the packets sent over each path. Hosts SHOULD stop
 sending traffic on paths if (a) no non-probing packet was received
-in the last 3 path RTTsor (b) no non-probing packet sent over this
+in the last 3 path RTTs or (b) no non-probing packet sent over this
 path was acknowledged during the last 3 path 
```

Comment:
I agree with the two simple ideas described above, I think it makes things much more clear.

Reply:
I also agree with the simpler ideas. The host should stop sending packets on a path if a) it experiences too many losses (i.e., they never got acknowledged), or b) a ABANDON_PATH frame has been sent/received concerning that path.

Reply:
@qdeconinck @huitema are you saying a node should not consider a path a closed if it was idle for RTTs? This was the original intent of the text.

Reply:
My point here is to have clear signals. I think we could leave an "idle timer" per path, but idle means "no sent packets acknowledged not received packets" on that path, and the duration of the path idle timer should have been negotiated (e.g., we could reuse the max_idle_timeout transport parameter per path, or define a new one if needed).

Reply:
Reusing the negotiated idle timer is the simpler solution for now.

Reply:
This is linked to a previous discussion on issue #51 . At that time, we were wondering whether to we should use max_idle_timeout or define something like max_path_idle_timeout. The max_idle_timeout in RFC9000 is used to close the whole connection, so maybe we want something per-path?

Reply:
Defining an idle timer per path would make control more precise. It would also be one more parameter that administrators will have to worry about. How do you set up this new parameter? Should it be the same for every path? How would it relate to the globally negotiated idle timer? I am not sure that the complexity is worth it. If a node wants to abandon a path before the globally defined timer expires, they can always explicitly abandon the path before the timer expires. I would rather keep it simple and have just one timer.

Reply:
I think, for now, we can replace the 3-RTT with the max_idle_timeout as Mirja @mirjak proposed. Regarding whether we want something per-path, if the timeout value is large enough, I prefer what Christian @huitema suggested that we have one timer. However, if we want a shorter timeout (something like 3 times RTT), then we might want to have a per-path timer because of path heterogeneity (For example, a 5G path's RTT could be much smaller than that of the other paths).

Reply:
We can keep most of the max_idle_timeout definition, just provide an suggested version for the replacement of the origin paragraph without : 

"Endpoints use the max_idle_timeout transport parameter to advertise 
the idle timeout of each paths. When more than one path is available, 
endpoints shall monitor the arrival of non-probing packets on 
the available paths. Endpoints SHOULD set the idle timers per path. 
An endpoint restarts its path idle timer when a packet from its peer
is received and processed successfully on this path, and endpoint also 
restarts its idle timer when sending an ack-aliciting packet if no other
ack-eliciting packets have been sent since last receiving and processing 
a packet on the current path. Endpoints SHOULD stop sending traffic 
on the path if the path is idle timeout, but MAY ignore that rule 
if it would disqualify all available paths. 

To avoid idle timeout of a path, endpoints can
send ack-eliciting packets such as packets containing PING frames
{{Section 19.2 of QUIC-TRANSPORT}} on that path to keep it alive.
Sending periodic PING frames also helps prevent middlebox timeout,
as discussed in {{Section 10.1.2 of QUIC-TRANSPORT}}.

Notice that one path may become unavailable before the path idle timeout, 
endpoints can send a PING or other ack-eliciting frame to test a path 
for liveness following {{QUIC-TRANSPORT}}. An endpoint SHOULD send 
PATH_ABANDON frame if it wants to stop using the path immediately, 
instead of waiting for the path idle timeout."

Reply:
@Yanmei-Liu not sure where you propose to add this text. Can you create a PR?